searx/translations/messages.pot

```diff
@@ -651,6 +651,10 @@ msgstr ""
 msgid "Reliability"
 msgstr ""
 
+#: searx/templates/simple/preferences/engines.html:23
+msgid "Weight"
+msgstr ""
+
```

Comment:
Can you remove the patch of the .pot file from this commit / thanks!

----

No need to extract translations; we have a [workflow](https://github.com/searxng/searxng/blob/733b795d5327a1be133365d9d46c41d709719109/.github/workflows/integration.yml#L163-L167) for the extraction.  This workflow is triggered when a patch (PR) is merged on branch master. 

Our translation process is more or less completely automated.  At first glance it looks a bit complicated, but it has the advantage that we have completely separated the translation from the master branch and changes to translations are incorporated into the normal review process via automated PRs.  The PRs from the translation tool are created once a week .. the entire process is described and illustrated here:
https://docs.searxng.org/dev/translation.html#searxng-weblate-workflow

Reply:
Oh right, that's cool!
I thought they would need to become added once to the root `messages.po` file and will then be added to the other languages via a workflow, but it's even better that way :)